src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -184,6 +184,8 @@ class DoorLockServer : public chip::app::AttributeAccessInterface
 
     inline bool SupportsFace(chip::EndpointId endpointId) { return GetFeatures(endpointId).Has(Feature::kFaceCredentials); }
 
+    inline bool SupportsAliro(chip::EndpointId endpointId) { return GetFeatures(endpointId).Has(Feature::kAliroProvisioning); }
```

Comment:
There is already a SupportsAliroProvisioning API. Cna you use that?

Reply:
Updated. Thank you